src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationClicktracking.java

```diff
@@ -0,0 +1,82 @@
+package com.wikia.webdriver.testcases.globalnavigationtests;
+
+import java.util.Arrays;
+import java.util.List;
+
+import javax.json.JsonObject;
+
+import org.testng.annotations.Test;
+
+import com.wikia.webdriver.common.clicktracking.ClickTrackingScriptsProvider;
+import com.wikia.webdriver.common.clicktracking.events.EventsGlobalNavigation;
+import com.wikia.webdriver.common.contentpatterns.SearchContent;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.global_navitagtion.NavigationBar;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+/**
+ * @author Michal 'justnpT' Nowierski
+ */
+public class GlobalNavigationClicktracking extends NewTestTemplate {
+
+    /** search test flow: action - expected event
+    *  navigate to main page
+    *  
+    *  search for query
+    *  click search button - search-button
+    *  click enter button - search-enter
+    *  
+    *  trigger suggestion - search-suggest-show
+    *  click enter on suggestion - search-suggest-enter
+    *  click enter after suggestion - search-after-suggest-enter
+    *  clear suggestion
+    *  
+    *  trigger suggestion
+    *  mouse click on suggestion - search-suggest
+    *  click search after suggestion - search-after-suggest-button
+    *  clear suggestion
+    */	
+    @Test(
+        groups = {"TestGlobalSearchInGlobalNav_001", "ClickTracking", "GlobalNav"}
+    )
+    public void search() {
```

Comment:
Really ? I will recheck it 
Thanks, put error message and environment configuration (where you runned it) 
I really don't want to push anything that fails :)